spark/README-short.txt

```diff
@@ -0,0 +1 @@
+Apache Spark™ is a multi-language engine for executing data engineering, data science, and machine learning on single-node machines or clusters.
```

Comment:
The short description for the Docker Hub is limited to 100 characters in a single line.